src/cinema/domainmodel/Seats.java

```diff
@@ -0,0 +1,29 @@
+package cinema.domainmodel;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class Seats {
```

Comment:
Class name `Seats` and it has field seats as well. It's a bit confusing. Could you think about another name for the class?

Reply:
outdated